setup.cfg

```diff
@@ -30,3 +30,11 @@ directory = htmlcov
 
 [flake8]
 ignore=D403,E226,E302,E305,E501,W503,W504
+
+[semantic_release]
+version_variable = rest_tools/__init__.py:__version__
```

Comment:
you need `lta` here

Reply:
also, the update to __init__.py's version tuple :smile:

Reply:
I pushed some changes for that. Does it look good now?

Reply:
looks good :+1: